.buildkite/_forge.rayci.yml

```diff
@@ -105,6 +105,19 @@ steps:
       - "3.11"
     env:
       PYTHON_VERSION: "{{matrix}}"
+      EXTRA_DEPENDENCY: core
+
+  - name: minbuild-others
+    label: "wanda: minbuild-{{matrix}}-py38"
+    wanda: ci/docker/min.build.wanda.yaml
+    depends_on: oss-ci-base_build
+    matrix:
+      - ml
+      - serve
+      - default
+    env:
+      PYTHON_VERSION: "3.8"
+      EXTRA_DEPENDENCY: "{{matrix}}"
```

Comment:
I really think that we should start moving test bases to their respective rayci files. the wanda steps do not have to stay in `_forge`.

we have too many of those in this file now, and they are really for all kinds of tests.

like if it is not a common one that is shared by multiple types of tests, but only used in one type, we should move the wanda step into those team level `rayci.yml` files

Reply:
good point, let me move these minbuild over, and other builds in another PR